client/src/pages/EquipmentBrowser/EquipmentBrowser.module.css

```diff
@@ -33,28 +27,85 @@
   background: rgb(171, 172, 231);
   justify-content: space-between;
   max-width: 65%;
-  flex-flow: row wrap;
 }
 
 .equipmentGrid > * {
   flex-flow: row-wrap;
   align-items: center;
 }
 
-.selectedEquipment {
+.cart {
   display: flex;
-  background: rgb(176, 177, 231);
+  background: white;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  color: #628aba;
+  justify-content: center;
   flex-grow: 1;
   flex-basis: 30%;
 }
 
 .collapsehead {
-  background: #add8e6;
-  margin-bottom: 10px;
-  border: 1px solid #0067cd;
+  background: #3279d5;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  padding: 6px;
+  color: white;
   cursor: pointer;
+  margin: 11px;
 }
```

Comment:
It would be great to add some more CSS here to visually refine the arrow icon a little bit more, specifically:
- vertically aligning it
- a little bit of animation?

I can suggest some CSS to add